### ISSUE TITLE
Create a "Windows Desktop application" for Visual Studio 2019

### ISSUE THREAD

**Issue by SuperDragonM** [MicrosoftDocs/cpp-docs#946]
I trying to follow "Walkthrough: Create a traditional Windows Desktop application (C++)" but is for Visual Studio 2017 and I using Visual Studio 2019, is`t hard to follow the tutorial, I can`t find "Windows application (.exe)" in "Windows Desktop Project" of the "Windows Desktop Wizard" I find only "Console application (.exe)" and "Desktop application (.exe)", so what should I choose?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: eda99bc7-483b-e53c-1035-886f4d0955f4
* Version Independent ID: a6fdba4d-53e4-9f1a-2352-bc68ef491069
* Content: [Walkthrough: Create a traditional Windows Desktop application (C++)](https://docs.microsoft.com/en-us/cpp/windows/walkthrough-creating-windows-desktop-applications-cpp?view=vs-2019)
* Content Source: [docs/windows/walkthrough-creating-windows-desktop-applications-cpp.md](https://github.com/Microsoft/cpp-docs/blob/master/docs/windows/walkthrough-creating-windows-desktop-applications-cpp.md)
* Product: **visual-cpp**
* Technology: **cpp-windows**
* GitHub Login: @mikeblome
* Microsoft Alias: **mblome**

**Comment by mikeblome** [comment]
@SuperDragonM apologies! You should choose Desktop Application (.exe). We are working as fast as we can to create new topics for Visual Studio 2019.  This planned work is on our backlog so I am closing this particular issue. Feel free to file a new issue if you encounter other problems.
